crates/swift-bridge-ir/src/parsed_extern_fn/to_swift_func.rs

```diff
@@ -39,11 +39,26 @@ impl ParsedExternFn {
                         todo!("Push to ParsedErrors")
                     };
 
-                    format!("{}: {}", arg_name, ty)
+                    if let Some(argument_labels) = &self.argument_labels {
+                        if let Some(argument_label) =
+                            argument_labels.get(&format_ident!("{}", arg_name))
+                        {
+                            params.push(format!(
+                                "{} {}: {}",
+                                argument_label.value().as_str(),
+                                arg_name,
+                                ty
+                            ))
+                        } else {
+                            let param = format!("{}: {}", arg_name, ty);
+                            params.push(format!("_ {}", param))
+                        }
+                    } else {
+                        let param = format!("{}: {}", arg_name, ty);
+                        params.push(format!("_ {}", param))
+                    }
                 }
             };
-
-            params.push(format!("_ {}", param))
```

Comment:
I think we can keep the `push` on the outside and just change it to `params.push(param)`.

Then in the match block we can put the `_` or the `argument_label`.